src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcQueryOptionsTest.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner.jdbc.it;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.fail;
+
+import com.google.cloud.spanner.IntegrationTest;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.jdbc.ITAbstractJdbcTest;
+import com.google.cloud.spanner.jdbc.JdbcSqlException;
+import com.google.cloud.spanner.jdbc.SpannerPool;
+import com.google.rpc.Code;
+import com.google.spanner.v1.ExecuteSqlRequest.QueryOptions;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * {@link QueryOptions} can be used with the JDBC driver on three different levels:
+ *
+ * <ol>
+ *   <li>Specify {@link QueryOptions} in environment variables (TODO: Set environment variables
+ *       runtime)
+ *   <li>Specify {@link QueryOptions} in the connection URL or by setting these using <code>
+ *       SET &lt;query_option&gt; = '&lt;value&gt;'</code> statements.
+ *   <li>Specify {@link QueryOptions} in a query hint (TODO: Check SQL syntax)
```

Comment:
Could these TODOs be resolved now or do they need to be done later?

Reply:
The first one was already done (although it depends on changes in the client library that have not yet been merged).

I've checked the syntax for query hints and added a test for that as well, and removed the TODO.